### ISSUE TITLE
The Preview column in the Active Auto Number view is not accurate

### ISSUE THREAD

**Issue by maichung77** [BESDev/Celedon-Autonumber-RTM#26]
If we use it conditional auto-numbering and there are multiple entries for the same entity and attribute based on the value of another option set field, then the Preview column in the view would be set as 555 for  the records that are not applicable.  This is misleading.  I understand the value is accurate when you open the record and view it in the form, but is there anyway you can make the Preview column value accurate in the view?

**Comment by RBellefond** [comment]
I agree with the above post about how I would also like to see the Preview Column in the view display the same value as what is seen if you open the record.  Seeing the 555 in the View is very confusing.
